### ISSUE TITLE
✏️ [ FEAT REQ ] Pls., support minSdk21, Android5.0

### ISSUE THREAD

**Issue by DI555** [octoshrimpy/quik#150]
**Describe the feature you'd like**

Pls., support minSdk21, Android5.0. (or, if not possible, at least minSdk22, Android5.1)

As i know, it was dropped since after 3.9.4 brunch, ..but may be possible to come up with it to be supported? 
We have a lots of cute devices still perfect working with Android5.0/5.1 !!!
Many thanks in advance!!!

**Comment by octoshrimpy** [comment]
I agree: lots of old tech is still plenty usable nowadays. Looking into it

**Comment by octoshrimpy** [comment]
Due to some dependencies, 23 is the lowest I can go right now. May be able to support down to 21 or even 20 in the future. I'm going to keep this issue opened to remind me of it.
